kanalony-storage-logic/src/main/scala/kanalony/storage/logic/Queries.scala

```diff
@@ -9,9 +9,11 @@ object Queries {
   object HourlyUserActivityPrtnEntryClstCountry extends HourlyUserActivityPrtnEntryClstCountryQuery
   object HourlyUserActivityPrtnEntryQuery extends HourlyUserActivityPrtnEntryQuery
   object HourlyUserActivityClstEntryQuery extends HourlyUserActivityClstEntryQuery
+  object DailyUserActivityPrtnEntryQuery extends DailyUserActivityPrtnEntryQuery
   val queries : List[IQuery] = List(
```

Comment:
Sounds like a good idea, maybe we can achieve this "for free" for all sub-classes of IQuery, by implementing this code in the trait?